src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
```

Comment:
The check for exactly 7 days is too specific and does not account for rentals longer than 7 days. The function should handle any number of days and apply discounts for rentals of 7 or more days.